packages/bundle-size/src/reporters/sampleReport.js

```diff
@@ -0,0 +1,41 @@
+const { emptyDiff } = require('../utils/compareResultsInReports');
```

Comment:
can we move this file under `__fixture__` folder as it's only purpose is for tests?

Reply:
Done in 859b054.